index.html

```diff
@@ -0,0 +1,290 @@
+      </div>
+      <div class="profesion">
+        <h2 class="heading-content">
+          Profesional Art <br />
+          Printing Data
+        </h2>
+        <p class="p-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language">
+          <a class="block-two" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+        <button class="project-button" href="#">See Project</button>
+      </div>
+      <div class="profesion">
+        <h2 class="heading-content">
+          Profesional Art <br />
+          Printing Data
+        </h2>
+        <p class="p-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language">
+          <a class="block-two" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+        <button class="project-button" href="#">See Project</button>
+      </div>
+      <div class="profesion">
+        <h2 class="heading-content">
+          Profesional Art <br />
+          Printing Data
+        </h2>
+        <p class="p-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language">
+          <a class="block-two" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+        <button class="project-button" href="#">See Project</button>
+      </div>
+      <div class="profesion">
+        <h2 class="heading-content">
+          Profesional Art <br />
+          Printing Data
+        </h2>
+        <p class="p-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language">
+          <a class="block-two" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+        <button class="project-button" href="#">See Project</button>
+      </div>
+    </section>
+    <footer>
+      <section class="about">
```

Comment:
Kindly add the background image that is in the figma design in the about section.
This is your design
<img width="517" alt="Screenshot 2022-12-02 at 14 43 41" src="https://user-images.githubusercontent.com/48546992/205286273-1c4213fb-5cf5-4988-81de-f8121e7359f2.png">
This is the figma design:
<img width="377" alt="Screenshot 2022-12-02 at 14 47 28" src="https://user-images.githubusercontent.com/48546992/205286309-ffde9388-2d08-43dd-b637-009dd4775b57.png">